db/builder.cc

```diff
@@ -215,8 +215,7 @@ Status BuildTable(
       meta->fd.file_size = file_size;
       meta->marked_for_compaction = builder->NeedCompact();
       assert(meta->fd.GetFileSize() > 0);
-      tp = builder
-               ->GetTableProperties();  // refresh now that builder is finished
+      tp = builder->GetTableProperties();  // refresh now that builder is finished
```

Comment:
BTW I think there is something wrong with your local diff-format, letter count should be capped beneath 80.

Reply:
FYI I'm using `git diff -U0 HEAD | clang-format-diff.py -p1 -i` with source file from https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/clang-format-diff.py.

Reply:
😨 Let me check it.